src/ints/int10_vesa.cpp

```diff
@@ -236,29 +240,30 @@ Bit8u VESA_GetSVGAModeInformation(Bit16u mode,Bit16u seg,Bit16u off) {
 	} else if (pageSize) {
 		pages = (vga.vmemsize / pageSize)-1;
 	}
-	var_write(&minfo.NumberOfImagePages, pages);
-	var_write(&minfo.ModeAttributes, modeAttributes);
-	var_write(&minfo.WinAAttributes, 0x7);	// Exists/readable/writable
+	minfo.NumberOfImagePages = pages;
+	minfo.ModeAttributes = host_to_le16(modeAttributes);
+	minfo.WinAAttributes = 0x7; // Exists/readable/writable
 
 	if (mblock->type==M_TEXT) {
-		var_write(&minfo.WinGranularity,32);
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).